bundles/org.openhab.binding.luxtronikheatpump/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -2059,6 +2059,13 @@
 		<state pattern="%.1f %unit%" min="30" max="65" step="0.5"></state>
 	</channel-type>
 
+	<channel-type id="temperatureHeatingFixedReturnTarget">
```

Comment:
Don;t want to be a party pooper here. Not sure how strict the maintainers hold on to the namingconvention as this binding is allready full of the old method, but it should be `temperature-heating-fixed-return-target` or similar.

Reply:
I probably shouldn't break this by cleaning up the naming convention for all current users. But I also don't think I should do this one different than the others. What do you suggest?

Reply:
@openhab/add-ons-maintainers have to decide.

Reply:
As a general rule the [naming convention](https://www.openhab.org/docs/developer/guidelines.html#naming-convention) only applies to new bindings. Existing bindings should stick to the naming convention already used by the binding to keep consistent naming within the binding.